### DISCUSSION TITLE
Downloaded image isn't showing up on the map

### DISCUSSION THREAD

**Discussion by rafiibnsultan** [opengeos/segment-geospatial#101] [Q&A] [answered]
Hello,
Trying to replicate the example, in this step:

m.layers[-1].visible = False
m.add_raster(image, layer_name="Image")
m

The downloaded image isn't showing up in the map, only the map is being shown while running it on my PC. However, in the google colab it is working fine. Can you suggest what could be the problem?

Output of my PC: https://drive.google.com/file/d/1Ac8r0aXp0u0Wt4GeG7ckG8E79kweb55X/view?usp=drive_link 
Output of Colab: https://drive.google.com/file/d/1Ku9QzrowH6FRMc3ursi4NZ4N1bYTs30U/view?usp=drive_link

**Comment by rafiibnsultan** [comment]
### Environment Information

-   samgeo version:  0.8.1
-   Python version: 3.9.16
-   Operating System: Ubuntu 22.04.2 LTS

**Comment by giswqs** [comment, marked as answer]
Try the following example. If it does not work, it is a localtileserver issue. You need to [open](https://github.com/banesullivan/localtileserver/issues) an issue on the localtileserver repo. 

https://localtileserver.banesullivan.com/
```python
from localtileserver import TileClient, get_leaflet_tile_layer, examples
from ipyleaflet import Map

# Create a TileClient from a raster file
# client = TileClient('path/to/geo.tif')
client = examples.get_san_francisco()  # use example data

# Create ipyleaflet TileLayer from that server
t = get_leaflet_tile_layer(client)
# Create ipyleaflet map, add tile layer, and display
m = Map(center=client.center(), zoom=client.default_zoom)
m.add_layer(t)
m
```
